doc/source/project/organization.rst

```diff
@@ -106,12 +106,13 @@ Pull requests changing or clarifying the procedures governing this repository:
   *-1* binding votes.
 - Should be titled with the prefix *[PROCEDURES]* and tagged with
   the *procedures* tag in Github.
-- Should not be modified once open. If changes are needed, the pull request
-  should be closed, re-opened with modifications, and votes reset.
 - Should be restricted to just modifying the procedures and generally should not
   contain code modifications.
 - If the pull request adds or removes *committers*, there must be a separate
   pull request for each person added or removed.
+- Amendments may be made to the pull request by pushing additional changesets
+  but doing so restarts the moratorium and invalidates any votes which have
```

Comment:
Unfortunately I still think that it is not clear in the current formulation (i.e. "requiring reaffirmation of those votes prior to merge" obviously does not apply to vetoes), allowing veto bypassing.
I suggest that invalidating only *+1* votes obtains cleanly your goal, since vetoes still need to be withdrawn by their casters author prior to merge.